src/index.html

```diff
@@ -5,8 +5,369 @@
+              <div class="stars-container__star"></div>
+            </div>
+            <div class="stars-container__reviews">
+              Reviews: 5
+            </div>
+          </div>
+          <div class="price">
+            <div class="price__name">Price:</div>
+            <div class="price__amount">$2,199</div>
+          </div>
+          <button type="button" class="button card__button">
+            <a href="#buy" class="button__link">buy</a>
+          </button>
+        </div>
+
+
+        <div class="card main__card">
+          <img
+            src="./images/imac.jpeg"
+            alt="APPLE A1419 iMac 27"
+            class="card__image"
+          >
+          <div class="card__description">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </div>
+
+          <div class="card__product-code">
+            Product code: 195434
+          </div>
+
+          <div class="stars-container card__stars-container">
+            <div class="stars-container__stars stars-container__stars--active-4">
+              <div class="stars-container__star"></div>
+              <div class="stars-container__star"></div>
+              <div class="stars-container__star"></div>
+              <div class="stars-container__star"></div>
+              <div class="stars-container__star"></div>
+            </div>
+            <div class="stars-container__reviews">
+              Reviews: 5
+            </div>
+          </div>
+          <div class="price">
+            <div class="price__name">Price:</div>
+            <div class="price__amount">$2,199</div>
+          </div>
+          <button type="button" class="button card__button">
+            <a href="#buy" class="button__link">buy</a>
+          </button>
+        </div>
+
+
+        <div class="card main__card">
+          <img
+            src="./images/imac.jpeg"
+            alt="APPLE A1419 iMac 27"
+            class="card__image"
+          >
+          <div class="card__description">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </div>
+
+          <div class="card__product-code">
+            Product code: 195434
+          </div>
+
+          <div class="stars-container card__stars-container">
+            <div class="stars-container__stars stars-container__stars--active-4">
+              <div class="stars-container__star"></div>
+              <div class="stars-container__star"></div>
+              <div class="stars-container__star"></div>
+              <div class="stars-container__star"></div>
+              <div class="stars-container__star"></div>
+            </div>
+            <div class="stars-container__reviews">
+              Reviews: 5
+            </div>
+          </div>
+          <div class="price">
+            <div class="price__name">Price:</div>
+            <div class="price__amount">$2,199</div>
+          </div>
+          <button type="button" class="button card__button">
```

Comment:
```suggestion
          <div class="price">
            <div class="card__price-name">Price:</div>
            <div class="card__price-amount">$2,199</div>
          </div>
          <button type="button" class="card__button">
```
you need two BEM blocks card and stars, this should be a part of card block